vault/overlays/nerc-ocp-infra/backup-job/toolkitpod.yaml

```diff
@@ -0,0 +1,45 @@
+# Used for inspecting PVC used for the job, or retrieving/restoring snapshots
+
+# To retrieve a snapshot, deploy this pod in vault backup namespace
+# Then set VAULT_ADDR, have the root token ready
+# `oc rsh` into the toolbox container and run `vault login`
+# Use the root token to login.
+# Find the latest snapshot in the /snapshots directory. This is the pvc where snapshots are stored.
+# Then run `vault operator raft snapshot restore -force snapshot-${LATEST_TIMESTAMP}.snap`
+# You can also download the snapshot to your local file system by running:
+# oc cp debug-pod:/snapshots/snapshot-${LATEST_TIMESTAMP}.snap snapshot-${LATEST_TIMESTAMP}.snap
+# And restore vault via vault CLI from your local file system.
+apiVersion: v1
+kind: Pod
+metadata:
+  name: backup-toolkit
+spec:
+  serviceAccountName: backup-job
+  containers:
+    - name: toolbox
+      image: quay.io/operate-first/opf-toolbox:v0.8.0
+      command:
+        - sleep
+        - "36000"
```

Comment:
Why 36000? If this meant to hang around forever, then `sleep inf` is what you want.

Reply:
updated